design/ceph/ceph-mon-pv.md

```diff
@@ -37,6 +36,19 @@ template, then Rook will use a default value. This is possible because unlike
 the storage requirements of OSDs (xf: StorageClassDeviceSets), reasonable
 defaults (e.g. 5-10 GB) exist for monitor daemon storage needs.
 
+`MON_DISK_BIG` alert is triggered if the size of the monitor’s database is larger
+than `mon_data_size_warn` (default: 15 GiB) config. `DataSizeWarn` can be used to
+override the default `mon_data_size_warn`. `DataSizeWarn` should be used only in
```

Comment:
Instead of only restricting this feature to mons on PVCs, how about this:
- If `mon.dataSizeWarn` is specified, it will always be applied. 
- If `mon.dataSizeWarn` is not specified:
   - If on dataDirHostPath do nothing
   - If on a PVC, set the 90% default compared to the size of the PVC